tap_hubspot/hubspot.py

```diff
@@ -78,6 +78,8 @@ def streams(self, start_date: datetime, end_date: datetime):
             yield from self.get_archived("contacts")
         elif self.tap_stream_id == "archived_companies":
             yield from self.get_archived("companies")
+        elif self.tap_stream_id == "archived_deals":
+            yield from self.get_archived("deals")
```

Comment:
could I suggest doing a little less abstracting, ie. creating a function per [company|deals|contacts] called `get_archived_[company|deals|contacts]` that _then_ calls the `get_archived`. I know it is an indirection, but then we have a clear function on the hubspot client for the specific stream, meaning you don't have to go find it at the top all the time.

```

```

Reply:
Will fix